<launch>
  <node pkg="qualification" type="run_selftest.py">
        <remap from="node_name" to="imu"/>
  </node>
</launch>